chatkit-core/src/main/kotlin/com/pusher/chatkit/cursors/api/CursorSubscriptionEvent.kt

```diff
@@ -0,0 +1,8 @@
+package com.pusher.chatkit.cursors.api
+
+internal sealed class CursorSubscriptionEvent {
+    data class OnCursorSet(val cursor: CursorApiType) : CursorSubscriptionEvent()
+    data class InitialState(val cursors: List<CursorApiType>) : CursorSubscriptionEvent()
```

Comment:
good opportunity to put InitialState first as we have for other subscriptions
```patch
Index: chatkit-core/src/main/kotlin/com/pusher/chatkit/cursors/api/CursorSubscriptionEvent.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- chatkit-core/src/main/kotlin/com/pusher/chatkit/cursors/api/CursorSubscriptionEvent.kt	(revision 27a2db040d9d797c78012e598670919da8f7f803)
+++ chatkit-core/src/main/kotlin/com/pusher/chatkit/cursors/api/CursorSubscriptionEvent.kt	(date 1582123131063)
@@ -1,8 +1,8 @@
 package com.pusher.chatkit.cursors.api
 
 internal sealed class CursorSubscriptionEvent {
-    data class OnCursorSet(val cursor: CursorApiType) : CursorSubscriptionEvent()
     data class InitialState(val cursors: List<CursorApiType>) : CursorSubscriptionEvent()
+    data class OnCursorSet(val cursor: CursorApiType) : CursorSubscriptionEvent()
 }
 
 internal typealias CursorSubscriptionConsumer = (CursorSubscriptionEvent) -> Unit

```

Reply:
👍 makes sense! Have updated.